include/hc.hpp

```diff
@@ -1453,15 +1456,14 @@ accelerator_view::create_marker(memory_scope scope) const {
     std::shared_ptr<Kalmar::KalmarAsyncOp> deps[1]; 
     // If necessary create an explicit dependency on previous command
     // This is necessary for example if copy command is followed by marker - we need the marker to wait for the copy to complete.
-    hc::memory_scope depReleaseScope = hc::no_scope;
-    std::shared_ptr<Kalmar::KalmarAsyncOp> depOp = pQueue->detectStreamDeps(hcCommandMarker, nullptr, &depReleaseScope);
+    std::shared_ptr<Kalmar::KalmarAsyncOp> depOp = pQueue->detectStreamDeps(hcCommandMarker, nullptr);
```

Comment:
@bensander this is one of the greatest departure from the previous implementation. could you help clarify why we don't need `depReleaseScope` here, and why we don't need `greater_scope()` later in this function?